styleguide/source/_patterns/03-templates/02-ec-home/01-ec-home-three-up/ec-home-three-up.twig

```diff
@@ -0,0 +1,50 @@
+{% include 'organisms-ribbon' %}
+{% include 'molecules-alert-info' %}
+
+<div class="ec">
+    <section class="ec_search-bar">
```

Comment:
As noted above, `<section class="ec_search-bar">` should be part of the ec-search-bar pattern.